src/pudl/transform/eia930.py

```diff
@@ -0,0 +1,134 @@
+"""Module to perform data cleaning functions on EIA930 data tables."""
+
+import pandas as pd
+from dagster import AssetOut, Output, asset, multi_asset
+
+import pudl
+from pudl.metadata.enums import EIA930_GENERATION_ENERGY_SOURCES
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@multi_asset(
+    outs={
+        "core_eia930__hourly_balancing_authority_net_generation_by_energy_source": AssetOut(
+            io_manager_key="parquet_io_manager"
+        ),
+        "core_eia930__hourly_balancing_authority_operations": AssetOut(
+            io_manager_key="parquet_io_manager"
+        ),
```

Comment:
I went ahead and re-organized these tables into net generation by energy source and totals for generation, interchange, and demand as we discussed.